### PR TITLE
Adding ghostpii encryption package

### PR DESCRIPTION
**Pull request by jackp97**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jackp97** [comment]
@conda-forge/help-python ready for review

File: recipes/ghostpii/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ghostPii-{{ version }}.tar.gz
+  sha256: f67c0b9150a55e3b14cbfd4bec5ece848a13a74d79ba247a6dcad0969e3f24f9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```
Comment (carterbox) [inline recipes/ghostpii/meta.yaml:18]:
```suggestion
  host:
     - setuptools >=42
```

File: recipes/ghostpii/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+    - python >=3.6
+  run:
+    - IPython
```
Comment (carterbox) [inline recipes/ghostpii/meta.yaml:22]:
```suggestion
```
This package is not listed in requirements upstream.

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/ghostpii/meta.yaml (hunk 2 above)
Comment (carterbox) [reply recipes/ghostpii/meta.yaml]: Apparently IPython is imported, [but not listed in the requirements upstream](https://github.com/capnion/ghostpii_client/blob/010bf3a6c2e61719ee5e13f2b776c3f13b27625e/setup.cfg#L22). This is a bug that should be fixed upstream.

**Pull request merged**